app/assets/stylesheets/application.scss

```diff
@@ -333,19 +341,11 @@ table {
 
 .ellipsis {
   overflow: hidden;
+  overflow-wrap: break-word;
   white-space: nowrap;
   text-overflow: ellipsis;
 }
 
-.spinner-placeholder {
-  width: 16px;
```

Comment:
The spinner-placeholder function still exists just modified it a bit, we got the PF spinner class built in, so this css class is no longer needed.